static/js/rendered_markdown.js

```diff
@@ -235,6 +235,24 @@ export const update_elements = (content) => {
         });
     });
 
+    // Still image of the GIF is stored as a query in the href by
+    // our giphy.js module.
+    // We display the still image normally but show the GIF on hover.
+    content.find("a[href*=zulip_giphy_still_image]").each(function () {
+        const $a_gif = $(this);
+        const gif_url = $a_gif.attr("href");
+        const still_image_url = new URLSearchParams(gif_url).get("zulip_giphy_still_image");
+        $a_gif.find("img").attr("src", still_image_url);
+        $a_gif.closest(".messagebox-content").hover(
```

Comment:
Do we want this on message hover or image hover? I would prefer message hover.

Reply:
Hmm, good question!  I'd post the question in chat.zulip.org, but I think my intuition is probably message hover.  (And you can click the lightbox if you want to see it big -- we should make sure we've tested that this doesn't break that flow).

Reply:
Okay

Reply:
I think from a code structure perspective, I'm not sure I want this function to declare handler functions like the `hover` attribute; I'd prefer to just attach `data-still-image-src` type attributes and attach a CSS class and have a global function for that part.